<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_plot</class>
 <widget class="QDialog" name="Dialog_plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">	background-color: rgb(225, 225, 225);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(36, 36, 36);</string>
   </property>
   <property name="text">
    <string>Необходимо задать область определения по оси X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(36, 36, 36);</string>
   </property>
   <property name="text">
    <string>от</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ot_x">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Sans</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border:  solid gray;
background-color : white;
color: rgb(36, 36, 36);
border-radius: 5px;
}

</string>
   </property>
   <property name="text">
    <string>-10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="do_x">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Sans</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border:  solid gray;
background-color : white;
color: rgb(36, 36, 36);
border-radius: 5px;
}

</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(36, 36, 36);</string>
   </property>
   <property name="text">
    <string>до</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(36, 36, 36);</string>
   </property>
   <property name="text">
    <string>до</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(36, 36, 36);</string>
   </property>
   <property name="text">
    <string>и область значений по оси Y</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ot_y">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Sans</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border:  solid gray;
background-color : white;
color: rgb(36, 36, 36);
border-radius: 5px;
}

</string>
   </property>
   <property name="text">
    <string>-10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(36, 36, 36);</string>
   </property>
   <property name="text">
    <string>от</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="do_y">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Sans</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border:  solid gray;
background-color : white;
color: rgb(36, 36, 36);
border-radius: 5px;
}

</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(72, 148, 239);
  border: solid black;
	font: 17pt &quot;&quot;;
	font: 18pt &quot;PT Sans&quot;;
color: rgb(253, 253, 253);
border-radius: 17px;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(91, 169, 232), stop: 1 rgb(67, 122, 231));
}</string>
   </property>
   <property name="text">
    <string>Draw a graph</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
